app/views/find/result_filters/provider/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= content_for :page_title, "Pick a provider" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l" data-qa="heading">Pick a provider</h1>
+    <p class="govuk-body">You searched for ‘<%= params["query"] %>’</p>
+    <p class="govuk-body">We found these providers which matched your search:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <% @provider_suggestions.each do |provider| %>
+        <li data-qa="provider_suggestion">
+          <%= govuk_link_to(
+            "#{provider.provider_name} (#{provider.provider_code})",
+            start_subject_path(filter_params_without_previous_parameters.merge(query: provider.provider_name)),
+            { data: { qa: "provider_suggestion__link" } },
+          ) %>
+        </li>
+      <% end %>
+    </ul>
+
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+    <div>
+      <%= govuk_details(summary_text: "Try another provider") do %>
+        <%= form_with url: provider_path, method: :get, data: { "ga-event-form" => "Provider" } do |form| %>
```

Comment:
`provider_path` looks wrong...

Reply:
Yup the root wasn't there, doesn't do anything at the moment so have just added it in to use the correct path